src/components/provider/show/provider-show.js

```diff
@@ -73,6 +76,8 @@ class ProviderShow extends Component {
     };
   }
 
+  editPermissionName = 'ui-eholdings.records.edit';
```

Comment:
Same problem with naming
```suggestion
  RECORDS_EDIT_PERMISSION_NAME = 'ui-eholdings.records.edit';
```